app/javascript/crayons/Dropdown/Dropdown.jsx

```diff
@@ -1,11 +1,44 @@
 import { h } from 'preact';
+import { useLayoutEffect, useState } from 'preact/hooks';
 import PropTypes from 'prop-types';
 import { defaultChildrenPropTypes } from '../../common-prop-types/default-children-prop-types';
+import { initializeDropdown } from '@utilities/dropdownUtils';
+
+export const Dropdown = ({
```

Comment:
Refactor (blocking): The new props are not defined in the prop types. I wonder why the linter is not picking that up. 🤔